code/src/main/java/com/onlinecontacttracing/servlets/get-data/CalendarDataForPositiveUser.java

```diff
@@ -0,0 +1,15 @@
+package com.onlinecontacttracing.authentication;
+
+import com.google.api.client.auth.oauth2.Credential;
+
+class CalendarDataForPositiveUser implements Runnable {
```

Comment:
So, I still have an underlying question.

Will there be any difference between getting calendar data for a Positive User versus a Negative User? If so, what is it and why is the distinction needed?

If not, merge these.

Reply:
Yeah, will copy and paste my previous comment since it seems like it got lost in the process

Also this is an attempt at modularity, right now we could do it without a runnable but we could only handle one API, if later we wanted to handle two APIs, for example the Drive API, we would have to make two runnables and move all the code into the runnable.

If this is not clear enough I can elaborate.

Reply:
So, that seems to not answer my question. That seems to be answering why it is Runnable. 

The question is, we have a CalendarDataForPositiveUser and one for CalendarDataForNegativeUser. What's the difference between getting the data for these two different types of users? It seems like they both should be the same Calendar call.

Reply:
Oh no! I copied the wrong message

Reply:
I edited my response with the correct message

Reply:
If I understand correctly, we actually store data to storage in these runnables, right? If so that could explain the differentiation.

Reply:
Wait, so all PositiveUsers are stored in one object, while NegativeUsers are stored one user per object?

Reply:
If so, that's fine with me. It's difficult to see what's going on here with the skeletons, but if they are actually going to be different, that's that.